ontoportal

```diff
@@ -0,0 +1,238 @@
+#!/usr/bin/env bash
+
+run_api(){
+	clone_ongologies_linked_data(){
+	  repo_dir="ontologies_linked_data"
+
+	  if [ -d "$repo_dir" ]; then
+	      echo "[+] Directory $repo_dir already exists. Skipping cloning."
+	  else
+	    echo "[+] Cloning ontologies_linked_data repo..."
+	    git clone -q --depth=1 https://github.com/ontoportal-lirmm/ontologies_linked_data.git
+	  fi
+
+	  echo "[+] Generating Solr configsets ..."
+	  if [ -d "$repo_dir" ]; then
+	    cd ontologies_linked_data || exit 1
+	    if ! ./test/solr/generate_ncbo_configsets.sh; then
+	      echo  "[-] Error: Failed to generate Solr configsets."
+	      exit 1
+	    else
+	      echo "[+] Success: Generating Solr configsets."
+	    fi
+	    cd ..
+	  else
+	    echo "[-] Error: directory ontologies_linked_data does not exists"
+	  fi
+	}
+
+	clone_ongologies_linked_data
+	source .env
+	api_url="$API_URL"
+	if [ -z "$api_url" ] ; then
+		echo "[-] Error: Missing required arguments. Please provide both --api-url or update them in your .env"
+		exit 1
+	fi
+	bash_cmd=" (bundle check || bundle install || bundle update) && bundle exec rackup --host 0.0.0.0  --env=development --port 9393"
```

Comment:
if you remove `(bundle check || bundle install || bundle update)`  does it work ?